### DISCUSSION TITLE
能否区分一下针对同一视频不同分 p 和不同视频的倍速记忆

### DISCUSSION THREAD

**Discussion by liltigerzheng** [the1812/Bilibili-Evolved#2951] [功能建议 / Ideas]
希望能有一个开关来区分这两个
一个典型的使用场景是看网课的时候需要开倍速, 希望每次打开这个视频以及自动播下一 p 的时候都能记住倍速, 同时希望打开其他视频的时候是一倍速
当前只要在一个视频里调了倍速默认打开新视频也是这个倍速了...

**Comment by LonelySteve** [comment]
其实从 v1 开始，直到现在这个版本都是有这个功能的，只是选项被隐藏了，需要在控制台（`Ctrl + Shift + I`）里手动开启：

```javascript
bilibiliEvolved.settings.userComponents.rememberVideoSpeed.settings.options.individualRemember = true
```

如果希望调整打开其他视频时的默认速度（本质上是修改不启用独立记忆时的记忆倍速）：

```javascript
bilibiliEvolved.settings.userComponents.rememberVideoSpeed.settings.options.speed = 1
```

需要清除对当前视频的倍速记忆状态的话，使用快捷键（`Shift + :`）：
![image](https://user-images.githubusercontent.com/34429322/151644081-7075d4bb-817e-4eb8-984f-bf44a583094b.png)

这个功能是实验性的，加上从 v1 迁移过来没有仔细验证，可能不是特别稳定，等下一版出来，我会给它在界面上加上选项的。
